.github/workflows/report-vulnerability-in-gh.yaml

```diff
@@ -0,0 +1,78 @@
+name: Report vulnerability issues in Github
+on:
+  workflow_call:
+    inputs:
+      issue-title:
+        description: The title of the issue to be created/edited
+        required: true
+        type: string
+      issue-labels:
+        description: A comma separated list of labels
+        required: false
+        type: string
+        default: "bug"
+      image-name:
+        description: "Name of the oci-image as saved in Dockerhub or in the docker cache.
+          It consists of <rock-name>:<rock-version>."
```

Comment:
Does this work only for rocks?

Reply:
Actually is the container image name, not the rock, but it does consist of the rock name and the rock version as taken from the rockcraft.yaml file. Do you think the description can be clearer?

Reply:
Is this still true? I see https://github.com/DnPlas/kubeflow-rocks/issues?q=is%3Aopen+is%3Aissue issue include also the `tag` and IIUC it [uses](https://github.com/canonical/charmed-kubeflow-workflows/pull/73/files#diff-dd35b90c2446ee9fa26f8d74002a710fdda5ef2a01bc1e337971ce690d2e9551R25-R30) this workflow. If what I mentioned stands, then we could use `<image-name>:<image-tag>`, right?

Reply:
Yes comment below confirms that now we use `<image>:<tag>` https://github.com/canonical/charmed-kubeflow-workflows/pull/72#discussion_r1795759030